figures/pipeline/site_daily_metrics.py

```diff
@@ -88,6 +88,32 @@ def get_previous_cumulative_active_user_count(site, date_for):
         return 0
 
 
+def get_site_active_learners_for_date(site, date_for):
+    """
+    Get the active learners ids for the given site and date
```

Comment:
`Get the active learners id's for the provided site and date.`

Reply:
Done